.misc/deploy.pypi.sh

```diff
@@ -11,11 +11,27 @@ echo "Uploading coala to pypi"
 pip3 install twine wheel
 python3 setup.py sdist bdist_wheel
 # Upload one by one to avoid timeout
-# Don't block merging because pypi is in error 500 mood again
-set +e
-twine upload dist/*.whl -u "$PYPIUSER" -p "$PYPIPW"
-twine upload dist/*.tar.gz -u "$PYPIUSER" -p "$PYPIPW"
-set -e
+
+error=$(twine upload dist/*.whl -u "$PYPIUSER" -p "$PYPIPW" 2>&1 1>/dev/null)
+echo "$error"
```

Comment:
What I have done here is wrong.
But the problem if I pipe into `tee` is that `$?` gets the value of tee.
So the exception will not be caught.
I'll put the `echo` after the `if` instead